### DISCUSSION TITLE
libvips  on ubunthu 16.04 installation failed

### DISCUSSION THREAD

**Discussion by dhana1972** [libvips/libvips#4227] [Help wanted]
curently installed version
vips-8.2.2-Sat Jan 30 17:12:08 UTC 2016
ubunthu 16.04 
need to upgrade as below
image_processing/vips requires libvips 8.6+

**Comment by lovell** [comment]
Ubuntu Linux 16.04 reached end-of-life in 2021 and is unsupported, please upgrade.
